nist.c

```diff
@@ -0,0 +1,73 @@
+#include <oqs/common.h>
+#include <stdio.h>
+#include "api.h"
+#include "params.h"
+#include "nist_params.h"
+#include "xmss.h"
+
+int crypto_sign_keypair(unsigned char *pk, unsigned char *sk)
+{
+    xmss_params params;
+    uint32_t oid;
+    int ret = 0;
+
+    ret |= XMSS_STR_TO_OID(&oid, XMSS_OID);
+    if (ret)
+    {
+        printf("Did not recognize %s!\n", XMSS_OID);
```

Comment:
"Could not convert %s to oid" and maybe the macro should be XMSS_OID_STR since it's not an oid?

Reply:
Oh, I think the style is func_name(output, intput). 
I didn't change XMSS_STR_TO_OID macro because it's consistent with their code.